radlab-ui/webapp/src/components/modules/ModuleDeployment.tsx

```diff
@@ -86,116 +102,230 @@ const ModuleDeployment: React.FC<ModuleDeploymentProps> = ({
+            setDeploymentId((oldArray) => [
+              ...oldArray,
+              deployment.deploymentId,
+            ])
+          })
+      : setDeploymentId([])
+  }
+
+  const handleClick = (state: boolean) => setModal(state)
+
+  const renderModal = () => {
+    return (
+      <DeleteDeploymentModal
+        handleClick={handleClick}
+        deploymentIds={deploymentId}
+        handleRefresh={handleRefresh}
+      />
+    )
+  }
+
+  const handleCheckID = (id: string) => {
+    setAllSelect(false)
+    if (deploymentId.find((deployment_id) => deployment_id === id)) {
+      const value = deploymentId.filter((item) => item !== id)
+      setDeploymentId(value)
+    } else {
+      setDeploymentId((prev) => [...prev, id])
+    }
+  }
+
+  useEffect(() => {
+    deployments &&
+      deployments.map((deployment) => {
+        if (deployment.hasOwnProperty("deletedAt")) {
+          //@ts-ignore
+          deployment["status"] = "DELETED"
+        }
+      })
+    const value = deployments?.filter(
+      (deployment) => deployment.status === DEPLOYMENT_STATUS.SUCCESS,
+    )
+    if (value && value.length) setIsCheckbox(value.length)
+  }, [deploymentId])
+
   return (
-    <div className="overflow-x-auto bg-base-100 border border-base-300 p-0">
-      <table className="w-full divide-y divide-base-200 border border-base-200 rounded-md">
-        <thead className="bg-base-200">
-          <tr>
-            {headers.map((header) => (
-              <th
-                key={header.label}
-                scope="col"
-                className="px-4 py-3 text-xs md:text-sm font-medium text-left text-base-content tracking-wider"
-              >
-                <div className="relative">
-                  <div className="absolute -left-4 top-1">
-                    <ChevronUpIcon
-                      className={classNames(
-                        sortField === header.field &&
-                          sortDirection === SORT_DIRECTION.ASC
-                          ? "text-normal"
-                          : "hidden",
-                        "h-3",
-                      )}
-                    />
-
-                    <ChevronDownIcon
-                      className={classNames(
-                        sortField === header.field &&
-                          sortDirection === SORT_DIRECTION.DESC
-                          ? "text-normal"
-                          : "hidden",
-                        "h-3",
+    <>
+      <div className="flex justify-end mb-2">
+        <div
+          className="tooltip tooltip-primary"
+          data-tip="Select the modules to delete"
+        >
+          <button
+            className="btn btn-link btn-xs no-underline bg-base-200 hover:bg-base-300 flex gap-1 hover:no-underline"
+            disabled={!deploymentId.length}
+            onClick={() => setModal(true)}
+          >
+            <TrashIcon className="w-3 h-3" />
+            {t("delete")}
+          </button>
```

Comment:
Can we have this hidden completely until > 0 are selected. And can we move it to the bottom right as a `bg-error text-base-100` (white trash icon on red background) in a fixed position in the bottom right:
<img width="1147" alt="image" src="https://github.com/GoogleCloudPlatform/rad-lab/assets/69013624/3147e427-4c6b-41bc-a245-968fb06901de">